src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4673,6 +4690,10 @@ private void PreVisitMetadataArg(CallNode node, TexlFunction func)
 
                 // Typecheck the invocation and infer the return type.
                 fArgsValid = func.HandleCheckInvocation(_txb, args, argTypes, _txb.ErrorContainer, out returnType, out _);
```

Comment:
Same question for this line, what happens to the errors generated by this call?

Reply:
You are correct, there should have been temp error container here.